packages/env/index.js

```diff
@@ -2,4 +2,6 @@ const { EnvironmentPlugin } = require('webpack');
 
 module.exports = ({ config }, envs = []) => config
   .plugin('env')
-  .use(EnvironmentPlugin, ['NODE_ENV', ...envs]);
+  .use(EnvironmentPlugin, Array.isArray(envs) ?
+    ['NODE_ENV', ...envs] :
+    [{ NODE_ENV: 'development', ...envs }]);
```

Comment:
Since the `envs` value is an array, we should be spreading into the array instead of the object:

```js
[{ NODE_ENV: 'development' }, ...envs]);
```

Reply:
Also, we should probably be using the value from `neutrino.options.env.NODE_ENV`.

Reply:
Sorry for the late reply, crazy week.
I'll make the changes ASAP.

Reply:
Older Node doesn't allow spreading into objects, but does allow arrays, at least for the Node versions that Neutrino supports.

Doesn't an array of objects support both, by allowing multiple values with defaults?

```js
[
  // Pulls in NODE_ENV with the value defaulted to neutrino.options.env.NODE_ENV
  { NODE_ENV: neutrino.options.env.NODE_ENV },
  // Brings in any other envs, whether they are strings or objects
  ...envs
];
```